azure/function-app/storage_account.tf

```
@@ -1,5 +1,5 @@
 resource "azurerm_storage_account" "managed_storage_account" {
-  count = var.storage_account.name == null ? 1 : 0
+  count = var.storage_account.name == null ? 0 : 1
```

given that all other statements that are designed like this, in this repo work with `expression ? 1 : 0` 
maybe we stick with that, and alter the expression instead?
```suggestion
  count = var.storage_account.name != null ? 1 : 0
```